### DISCUSSION TITLE
How to use paraview software for visualizing the calculation results？

### DISCUSSION THREAD

**Discussion by yuantangjin** [christophmschaefer/miluphcuda#8] [Q&A] [answered]
Excuse me, I am having trouble visualizing the calculation results. I have calculated the Nakamura example and have got the complete result file. I see the ReadMe file in this example folder and I am following it. The **paraview** I use is **5.8.0.** After each operation: load the state file "Nakamura. Pvsm", a pop-up window will appear: load state options. I have tried every option in this pop-up window, and then click the OK option. Each time, the software will flash back. What is the reason? Is there something wrong with my operation? Can you give me some help? @christophmschaefer Thank you very much!

**Comment by christophb02** [comment]
Hi, in the "Load State Options" window you have to select "Choose File Names" and then (below) select the file paraview.xdmf  that you have just created for your results. Or does that not work?

**Comment by yuantangjin** [comment, marked as answer]
That's doesn't work, and the software will flash back, I don't know if it is the software version, so I would like to ask which version of the Paraview software you are using😅



---Original---
From: "Christoph ***@***.***&gt;
Date: Mon, Nov 8, 2021 22:51 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [christophmschaefer/miluphcuda] How to use paraview software for visualizing the calculation results？ (Discussion #8)




 
Hi, in the "Load State Options" window you have to select "Choose File Names" and then (below) select the file paraview.xdmf  that you have just created for your results. Or does that not work?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by christophb02** [reply]
The nakamura.pvsm file says it has been created with Paraview 5.7.0 (this has been some time ago). In my experience these state files are compatible over quite a range of Paraview versions, but obviously there are limits... please either try it with 5.7.0 if that's not much effort and acceptable for you, or alternatively you can always open the .xdmf file directly and then choose settings yourself. The latter is not too complicated, Paraview is pretty simple to use and has a nice and intuitive gui...!

**Reply by yuantangjin** [reply]
I'm glad to tell you that **my problem has just been solved.** It is confirmed that there is **no problem with the file I calculated and output**. The **problem** lies in the **version of the software**. I used version 5.8.0 before. I operate according to your readme file, and the software flashes out every time.Because I also consider that the higher version of software should be compatible with the lower version, and when I change the software version, it is difficult to download, so I have not changed the software version. At first, I thought there was a problem with the format of my calculation output. Now it seems that it is **the problem of paraview software**. I **haven't figured out the specific reason**. In short, it **can be visualized now**, ha ha, **Thank you very much** for your guidance @christophb02

**Reply by christophb02** [reply]
Good to hear that it is working now. May I ask how you got it running now? For the record...

**Reply by yuantangjin** [reply]
I just changed the version of the paraview software, and the problem was solved by changing to the 5.7.0 version. The steps are to follow the steps you gave.
